lib/rules/landmark-unique-matches.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Since this is a best-practice rule, we are filtering elements as dictated by ARIA 1.1 Practices regardless of treatment by browser/AT combinations.
+ *
+ * Info: https://www.w3.org/TR/wai-aria-practices-1.1/#aria_landmark
+ */
+const excludedParentsForHeaderFooterLandmarks = [
+	'article',
+	'aside',
+	'main',
+	'nav',
+	'section'
+].join(',');
+function isHeaderFooterLandmark(headerFooterElement) {
+	return !axe.commons.dom.findUpVirtual(
+		headerFooterElement,
+		excludedParentsForHeaderFooterLandmarks
+	);
+}
+
+function isLandmarkVirtual(virtualNode) {
+	const { actualNode } = virtualNode;
+	const landmarkRoles = axe.commons.aria.getRolesByType('landmark');
+	const role = axe.commons.aria.getRole(actualNode);
+	if (!role) {
+		return false;
+	}
+
+	const nodeName = actualNode.nodeName.toUpperCase();
+	if (nodeName === 'HEADER' || nodeName === 'FOOTER') {
+		return isHeaderFooterLandmark(virtualNode);
+	}
+
+	if (nodeName === 'SECTION' || nodeName === 'FORM') {
```

Comment:
I believe in those cases their roles would be form and section since they are explicit (unlike nav in that example, which is implicit).

If my understanding is incorrect, I can definitely make that change.